docs/.eleventy.js

```diff
@@ -91,6 +91,19 @@ module.exports = function(eleventyConfig) {
         return markdown.render(value);
     });
 
+    /*
+     * Removes `.html` suffix from the given url.
+     * `page.url` will include the `.html` suffix for all documents
+     * except for those written as `index.html` (their `page.url` ends with a `/`).
+     */
+    eleventyConfig.addFilter("prettyURL", url => {
```

Comment:
I’m concerned about this approach. I don’t think it’s at all clear where we should and should not be using this filter, and I fear this will cause more errors in the future.

Reply:
The rule would be to use this filter in all places where `page.url` is written into the output content. In places where it is used in calculations, there is no need to use the filter ([example on the current website](https://github.com/eslint/website/blob/2894f93e3f720e82de372785ff1ec0d81accad58/_layouts/post.liquid#L54)). This is the same approach as on the current website.

If you have an idea for another approach, I could try it. We could maybe write the files without `.html` extension, but that would probably require additional configuration on Netlify to treat extensionless files as html.